src/Symfony/Component/BrowserKit/Client.php

```diff
@@ -322,18 +318,17 @@ public function submit(Form $form, array $values = array()/*, array $serverParam
     }
 
     /**
-     * Finds a form by submit button text and then submits it.
+     * Finds the first form that contains a button with the given content and
+     * uses it to submit the given form field values.
      *
-     * @param string $button The button text
-     * @param array  $values An array of form field values
-     * @param string $method The method for the form
-     *
-     * @return Crawler
+     * @param string $button      The text content, id, value or name of the form <button> or <input type="submit">
+     * @param array  $fieldValues Use this syntax: array('my_form[name]' => '...', 'my_form[email]' => '...')
+     * @param string $method      The HTTP method used to submit the form
      */
-    public function submitForm($button, $values, $method)
+    public function submitForm(string $button, array $fieldValues, string $method = 'POST'): Crawler
```

Comment:
What about having `array()` as the default value for `$fieldValues`?

Reply:
I like that! Done!